pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1260,53 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := c.nodeConfig.GatewayConfig.Name
```

Comment:
rename the `gateway` variable to `iface` as the "gateway" name is not appropriate for policyOnly mode

maybe also add a comment for future readability:

> the interface on which ICMP probes are sent / received is the Antrea gateway interface, except in networkPolicyOnly mode, for which it is the Node's transport interface.